src/common/commonutils/PackageUtils.c

```diff
@@ -178,6 +179,28 @@ int CheckPackageNotInstalled(const char* packageName, char** reason, void* log)
     return result;
 }
 
+void AptGetUpdateOnce(void* log)
+{
+    const char* command = "apt-get update";
+    int status = 0;
+    if (g_aptGetUpdateExecuted)
+    {
+        return;
+    }
+
+    status = ExecuteCommand(NULL, command, false, false, 0, 0, NULL, NULL, log);
+    if (0 == status)
+    {
+        OsConfigLogInfo(log, "AptGetUpdateOnce: apt-get update was successful");
+    }
+    else
+    {
+        OsConfigLogError(log, "AptGetUpdateOnce: apt-get update failed with %d", status);
+    }
+
+    g_aptGetUpdateExecuted = true;
```

Comment:
Should we set to true even though it fails on L198 so it can try again on a subsequent run?

Reply:
Sure, let's try that